lib/logitech_receiver/receiver.py

```diff
@@ -334,29 +334,31 @@ def __init__(self, handle, device_info):
 		self.path = device_info.path
 		# USB product id, used for some Nano receivers
 		self.product_id = device_info.product_id
+		product_info = _product_information(self.product_id)
+		if not product_info:
+			raise Exception("unknown receiver type", self.product_id)
 
 		# read the serial immediately, so we can find out max_devices
 		# this will tell us if it's a Unifying or Nano receiver
```

Comment:
Remove the comment please.

Reply:
You mean the one about reading the serial number?  The second line is incorrect, so I'm removing that.  The first line was there before and is still valid.  Should I remove that as well?

Reply:
No, that is fine.